pyproject.toml

```diff
@@ -176,12 +176,33 @@ select = [
   "F",   # pyflakes
   "PL",  # pylint
 ]
+preview = true
```

Comment:
What does `preview=true` actually mean?

Reply:
I tried to sort of write that in the git commit mesage. 

More information at https://docs.astral.sh/ruff/preview/

Reply:
Right, I see now - haven't checked the commit message :)